apps/web/src/utils/initCCA.ts

```diff
@@ -0,0 +1,43 @@
+// The CCA team said this lite version of the library is temporary and not officially supported.
+// They recommended disabling linting and type-checking for now, since this version is not typed.
+/* eslint-disable */
+// @ts-nocheck
+import getConfig from 'next/config';
+
+const { publicRuntimeConfig } = getConfig();
+
+// CCA library loads in _app.tsx
+const initCCA = (router) => {
+  if (window.ClientAnalytics) {
+    const { init, identify, PlatformName, initNextJsTrackPageview } = window.ClientAnalytics;
+
+    init({
+      isProd: process.env.NODE_ENV === 'production',
```

Comment:
We won't have access to `process.env.NODE_ENV`, U'll need to access that through publicRuntimeConfig as well